code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -473,6 +473,7 @@
 		/obj/item/rsf,
 		/obj/item/instrument/violin,

```

Comment:
with piano synth added this and the guitar are reduntant now

Reply:
maybe. but nothing will bring back the satisfaction of beating the captain to death with a guitar. maybe just the violin is a bit useless.

Reply:
oh fuck, you're absolutely correct

the guitar does make a unique sound upon hitting someone

Reply:
yeah fuck the violin.